setup.cfg

```diff
@@ -1,62 +1,5 @@
-[metadata]
-name = asdf_astropy
-author = The Astropy Developers
-author_email = astropy.team@gmail.com
-license = BSD-3-Clause
-license_file = LICENSE.rst
-url = https://github.com/astropy/asdf-astropy
-description = ASDF serialization support for astropy
-long_description = file: README.rst
-long_description_content_type = text/x-rst
-edit_on_github = False
-github_project = astropy/asdf-astropy
 
-[options]
-zip_safe = True
-packages = find:
-python_requires = >=3.8
-setup_requires = setuptools_scm
-install_requires =
-    #astropy @ git+https://github.com/astropy/astropy
-    astropy>=5.0.4
-    asdf>=2.8.0
-    numpy
-    asdf-coordinates-schemas
-    asdf-transform-schemas>=0.2.2
-    importlib_resources>=3;python_version<"3.9"
-    packaging>=16.0
-
-[options.entry_points]
-asdf.extensions =
-    asdf-astropy = asdf_astropy.integration:get_extensions
-asdf.resource_mappings =
-    asdf-astropy = asdf_astropy.integration:get_resource_mappings
-
-[options.extras_require]
-test =
-    pytest-astropy
-    coverage
-    scipy
-docs =
-    sphinx
-    sphinx-astropy
-    sphinx-automodapi
-    sphinx-asdf
-    graphviz
-    matplotlib
-    docutils
-
-[options.package_data]
-asdf_astropy.resources = manifests/*.yaml, schemas/*.yaml, schemas/**/*.yaml, schemas/**/**/*.yaml
-
-[tool:pytest]
-testpaths = "asdf_astropy" "docs"
-astropy_header = true
-doctest_plus = enabled
-text_file_format = rst
-addopts = --color=yes --doctest-rst
-asdf_schema_root = asdf_astropy/resources/schemas
-asdf_schema_tests_enabled = true
+# this file exists to support for `flake8` (https://github.com/PyCQA/flake8/issues/234)
```

Comment:
this file can be `.flake8` if we don't use `pyproject-flake8` (or similar package like `flakehell`, `flake9`, `flake8-pyproject`, etc.)